internal/log/processor.go

```diff
@@ -112,6 +113,8 @@ func (lp *Processor) Process() {
 				} else {
 					Spamf("Failed to parse esc sequence while processing logs: invalid log mode: %s", args[0])
 				}
+			case EscapeMarkerOff:
+				break
```

Comment:
This breaks only the nearest switch statement (so doesn't change anything). I don't think it's an outcome you had in mind.